### PR TITLE
[DE] HassGetWeather

### PR DESCRIPTION
**Pull request by steffenrapp**
German translation of #1627

### CODE REVIEW

File: sentences/de/weather_HassGetWeather.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+language: de
+intents:
+  HassGetWeather:
+    data:
+      - sentences:
+          - "wie ist [das] Wetter [(für|in) <name>]"
+          - "wie ist [das] <name> Wetter"
```
Comment (mib1185) [inline sentences/de/weather_HassGetWeather.yaml:7]:
```suggestion
          - "wie ist [das] <name>[er] Wetter"
```

what do you think about this addition? it should make "Wie ist das Berliner Wetter?" possible 🤔

Comment (steffenrapp) [reply sentences/de/weather_HassGetWeather.yaml:7]: I like that idea! 😃

Comment (steffenrapp) [reply sentences/de/weather_HassGetWeather.yaml]: I also added your test sentence

**Review by mib1185 — APPROVED** [review]
LGTM, thx @steffenrapp 👍

**Pull request merged**